cloudferrylib/os/identity/keystone.py

```diff
@@ -21,6 +21,7 @@
 
 import cfglib
 from cloudferrylib.base import identity
+from cloudferrylib.utils import retrying
```

Comment:
move this line to between 
```
from cloudferrylib.utils import proxy_client
from cloudferrylib.utils.utils import GeneratorPassword
```
to keep an order of imports according to general openstack rules: http://docs.openstack.org/developer/hacking/#imports